actionpack/lib/action_controller/railtie.rb

```diff
@@ -67,5 +67,12 @@ class Railtie < Rails::Railtie #:nodoc:
         config.compile_methods! if config.respond_to?(:compile_methods!)
       end
     end
+
+    initializer "setup session store" do |app|
+      ActiveSupport.on_load(:action_controller) do
```

Comment:
@rafaelfranca We can move it to Rails::Application::Configuration class. What do you think?